.github/workflows/container-image.yml

```diff
@@ -59,12 +75,21 @@ jobs:
       run: make build-client
 
   test-server:
-    #needs: build-server
+    needs: build-server
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+    - name: Retrieve image
+      uses: actions/download-artifact@v3
+      with:
+        name: server-image-artifact
+        path: /tmp
+    - name: Load image
+      run: |
+        docker load --input /tmp/server-image.tar
+        docker tag localhost/samba-container:latest samba-container:latest
```

Comment:
Why is the `docker tag ....` line necessary? Doesn't docker (and podman) treat `localhost/blah:foo == blah:foo` ?  If you tried it and it doesn't work, no worries. I'm just curious because I thought it did.

Reply:
Even I thought so but its not the case it seems.